frontend/src/data/dataStorage.ts

```diff
@@ -110,6 +111,9 @@ export const addQuiz = async (quiz: QuizWithOnlyBody): Promise<boolean> => {
     },
     imageURL: '',
   };
+  quizWithBody.answerVariants = quizWithBody.answerVariants.map(
```

Comment:
```suggestion
  const quizeNormalized = { ...quizWithBody, answerVariants: quizWithBody.answerVariants.map( 
```

Reply:
Thanks, fixed with commit: [fix addQuiz to recreate new object and not to modify old](https://github.com/d9nchik/kpi_student/pull/6/commits/b3e750743a203c714a32f23f38c4fd7d88544e81)